### ISSUE TITLE
Application layout doesn't accommodate tag logos

### ISSUE THREAD

**Issue by steverydz** [canonical/vanilla-framework#5014]
## Component/pattern to amend

The application layout

## Visual

<img width="377" alt="Screenshot 2024-02-29 at 15 45 55" src="https://github.com/canonical/vanilla-framework/assets/501889/8609e568-8943-4fa1-9096-768ba5a9c079">

<img width="457" alt="Screenshot 2024-02-29 at 15 47 53" src="https://github.com/canonical/vanilla-framework/assets/501889/5ee7946c-3520-4f5c-a767-90ed58e2d74a">

## Context

The application layout doesn't allow for latest logos which include a tag and are intended to have no space between them and the top of the screen.

There is also a related issue where in the examples the circle/icon is separate from the word mark which is for the unpinned navigation on narrow viewports. This doesn't work with this type of logo as the text isn't the same height of the logo tag/icon.

**Comment by bartaz** [comment]
Yes, that's long overdue. We had an attempt on addressing this (https://github.com/canonical/vanilla-framework/pull/4691), but it got blocked on an ongoing redesign/spec of the whole side navigation.

But I guess it's good to bring this back to attention and decide on WG how to proceed with this.

So far I think some apps (Anbox, LXD, etc) have some local solutions around this problem, so maybe this can be shared somehow.

**Comment by danielmutis** [comment]
WG: This issue is validated. We have some solutions in the new side nav spec, which hasn't been fully approved yet. This includes more changes than just the logo. For now, there will be some short-term solutions to this problem until we get a finalised side nav.

**Comment by bartaz** [comment]
Duplicate of #4711
